src/AppInstallerCLICore/ContextOrchestrator.cpp

```diff
@@ -23,6 +26,17 @@ namespace AppInstaller::CLI::Execution
                 queue->RunItem(queueItem->GetId());
             }
         }
+
+        // Get command queue name based on command name.
+        std::string_view GetCommandQueueName(std::string_view commandName)
+        {
+            if (commandName == "install" || commandName == "uninstall")

```

Comment:
These strings should come from the command via a static function rather than be copied here.